packages/core/sample-projects/src/actors/app.ts

```diff
@@ -9,7 +9,7 @@ interface HelloMessage {
 
 export type AppMessage = HelloMessage;
 
-@ActorProvider({ name: 'app' })
+@ActorProvider({ name: 'app' }, true)
```

Comment:
> It's hard to understand what `true` means

It means that this actor will be initiated and bound to the registry while the DApp is up.

Reply:
> > It's hard to understand what `true` means
> 
> It means that this actor will be initiated and bound to the registry while the DApp is up.

It can be understood by inspecting the source code. Can we make it clearer with a labeled key so readers can get it instantly?

Reply:
> > > It's hard to understand what `true` means
> > 
> > 
> > It means that this actor will be initiated and bound to the registry while the DApp is up.
> 
> It can be understood by inspecting the source code. Can we make it clearer with a labeled key so readers can get it instantly?

I didn't quite understand

Reply:
> > > > > It's hard to understand what `true` means
> > > > 
> > > > 
> > > > It means that this actor will be initiated and bound to the registry while the DApp is up.
> > > 
> > > 
> > > It can be understood by inspecting the source code. Can we make it clearer with a labeled key so readers can get it instantly?
> > 
> > 
> > I didn't quite understand
> 
> A field `(true)` doesn't deliver the usage of `true`, providing a `labeled key` would be informational, as `{ bind: true }`

I think the name of the parameter contains the same information

Reply:
But the parameter is inaccessible without a tool like a language server

Say I copy the code snippet to others, and a comment is necessary to deliver the meaning of `true`

Reply:
Indeed, it should also be refactored to make constructor parameters clear